apps/cowswap-frontend/src/modules/swap/containers/SwapWidget/propsChecker.ts

```diff
@@ -34,7 +21,7 @@ function isCurrencyInfoEqual(prev: CurrencyInfo, next: CurrencyInfo): boolean {
     isFiatAmountEqual &&
     isAmountEqual &&
     isIsIndependentEqual &&
-    isReceiveAmountInfoEqual(prev.receiveAmountInfo, next.receiveAmountInfo)
+    prev.receiveAmountInfo === next.receiveAmountInfo
```

Comment:
Isn't this comparing objects?

Reply:
It's not a problem anymore, because `receiveAmountInfo` is cached in jotai atom